dm/master/openapi.go

```diff
@@ -133,17 +133,85 @@ func (s *Server) DMAPIDeleteSource(ctx echo.Context, sourceName string) error {
 
 // DMAPIStartRelay url is:(POST /api/v1/sources/{source-id}/relay).
 func (s *Server) DMAPIStartRelay(ctx echo.Context, sourceName string) error {
-	return nil
+	var req openapi.StartRelayRequest
+	if err := ctx.Bind(&req); err != nil {
+		return err
+	}
+	sourceCfg := s.scheduler.GetSourceCfgByID(sourceName)
```

Comment:
duplicated check in `scheduler.StartRelay` and `scheduler.UpdateSourceCfg`

Reply:
we need to get the source cfg out in advance so we can check if we need to go to UpdateSourceCfg,

and i think this check(source cfg is exist) is also essential in the start/stop relay function